### ISSUE TITLE
Some kind of progress bar for the `static:warm` command

### ISSUE THREAD

**Issue by duncanmcclean** [statamic/ideas#696]
It would be nice if the `static:warm` command for warming the static cache had some kind of progress bar so we know how it's actually going on.

## Use case

We have a site with thousands of entries (& URLs) where it would be kinda nice to know how long through those thousands of entries it is.

**Comment by edalzell** [comment]
It shows what url it's generating, where would the progress bar go?

**Comment by duncanmcclean** [comment]
Yes, but you have no idea whether it is 10% through generating the site or 95%.

Which with a big site would be helpful to know since the site would be pretty slow (if not, useless due to the CPU usage) while generating.

**Comment by edalzell** [comment]
ya I think Arthur tried to add a counter but because it's done in a configurable pool, asynchronously, it didn't work well.

**Comment by edalzell** [comment]
but also, you watch your deployments like that?

**Comment by duncanmcclean** [comment]
I don't - our plan is to only clear the static cache in bits so we'll probably never clear & regenerate the whole thing. 

Although, with that thinking I probably don't need this FR 😅

**Comment by callum-veloxcommerce** [comment]
This would be nice for checking status of a large deployment
